arangod/Aql/Executor/IndexAggregateScanExecutor.h

```diff
@@ -61,6 +61,8 @@ struct IndexAggregateScanInfos {
   containers::FlatHashMap<VariableId, size_t> _expressionVariables;
 
   QueryContext* query;
+  ResourceMonitor* resourceMonitor;
+  std::unique_ptr<ResourceUsageScope> usageScope;
```

Comment:
Is there any memory that we retain for the full runtime of this Infos?
Or should the scope be a bit more local?
If the former is the case keep this, on the later please remove the usageScope.
ResourceMonitor can stay, however, it could be collected from the QueryContext on the fly as well so a helper function would suffice.